Core/Core/People/PeopleListViewController.swift

```diff
@@ -65,15 +69,27 @@ public class PeopleListViewController: UIViewController, ColoredNavViewProtocol
         tableView.backgroundColor = .named(.backgroundLightest)
         tableView.refreshControl = UIRefreshControl()
         tableView.refreshControl?.addTarget(self, action: #selector(refresh), for: .primaryActionTriggered)
+        tableView.registerHeaderFooterView(FilterHeaderView.self, fromNib: false)
         tableView.separatorColor = .named(.borderMedium)
 
         colors.refresh()
-        users.refresh()
         if context.contextType == .course {
             course.refresh()
         } else {
             group.refresh()
         }
+        users.refresh()
+
+        if context.contextType == .course {
+            env.api.makeRequest(GetConversationRecipientsRequest(search: "", context: context.canvasContextID, includeContexts: true)) { [weak self] recipients, _, _ in
```

Comment:
Is pagination going to be an issue here? This won't get all the recipients so the enrollment types array might be incomplete. You could use `env.api.exhaust` but that seems a bit heavy for large courses.

Reply:
this only returns the top level groups, no individuals, for a course, I've never seen more than 6 results.

Reply:
Okay that makes sense.